doc.php

```diff
@@ -47,7 +59,8 @@
         'anchor':     window.location.hash.substring(1),
         'branch':     '$branch',
         'repository': '$repository',
-        'url':        'https://raw.githubusercontent.com/$repository/webots-doc/'
+        'url':        'https://raw.githubusercontent.com/$repository/webots-doc/',
+        'rawgiturl':  '$rawgiturl'
```

Comment:
If I'm correct, this seems to be unused.

Reply:
it is unused, but I figured it would be useful for debugging purposes, no?

Reply:
I would say no: when we would like to debug, we have to introduce new variables etc., but introducing them in prevision to debug later is complexifying the code.